src/Microsoft.AspNetCore.Session/DistributedSession.cs

```diff
@@ -65,6 +71,33 @@ public class DistributedSession : ISession
             _isNewSessionKey = isNewSessionKey;
         }
 
+        public string Id
+        {
+            get
+            {
+                Load(); // TODO: Silent failure
+                if (_sessionId == null)
+                {
+                    _sessionId = new Guid(IdBytes).ToString();
+                }
+                return _sessionId;
+            }
+        }
+
+        private byte[] IdBytes
+        {
+            get
+            {
+                Load(); // TODO: Silent failure
+                if (_sessionIdBytes == null)
+                {
+                    _sessionIdBytes = new byte[IdByteCount];
+                    CryptoRandom.GetBytes(_sessionIdBytes);
```

Comment:
16 bytes of crypto random? Pretty low... We don't actually rely on it being unique for any functional purpose, we don't use it for anything.